lib/MailQueueHandler.php

```diff
@@ -411,6 +411,12 @@ protected function sendEmailToUser($userName, $email, $lang, $timezone, $maxTime
 		$message->useTemplate($template);
 		$message->setFrom([$this->getSenderData('email') => $this->getSenderData('name')]);
 
+		if(method_exists($message, 'setAutoSubmitted')){
```

Comment:
Activity is bundled/shipped with the server so once the server PR is merged, this is not needed.

Reply:
But we can copy the else branch to stable25 and previous as a backport

Reply:
Ok. And how would that work? creating new PRs against these older branches? or something else?

Reply:
That only makes sense if the server bit would be backported and I'm not sure it is, as it does not look like a critical nor security relevant thing